### DISCUSSION TITLE
Environment won't change!?

### DISCUSSION THREAD

**Discussion by LesBLUUs** [smartcontractkit/full-blockchain-solidity-course-py#1382] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/97270916/162548315-17a40fb2-0b0c-41b8-8af9-4eb2afcc7854.png)
Hello all, and thanks in advance. I've been looking struggling to figure out why my environment seems to be "stuck" in VM mode? I got prompted to connect my wallet and everything on my initial attempt to select Injected Web3 and the first time it changed as it should. However, it didn't load in my account that field was blank so I refreshed the page. Since the refresh the environment wont even change from Javascript VM to injected anymore!? For some odd reason the only change is that it says Rinkeby (4) network under the environment slot or main if I change it in my wallet but nothing else works. Can't see my account nor can I deploy the contract.

I've looked all over tried Brave and chrome....nothing works.

Any help is greatly appreciated!!

**Comment by cromewar** [comment, marked as answer]
Hello @LesBLUUs 
This is most likely a bug on the browser (brave and chrome have the same nucleus) so I have some recommendations:
1. Try with mozilla firefox.
2. Delete the entire, cache, cookies and historial of the brownie and try gain. (take in mind this will delete your contracts)

**Reply by LesBLUUs** [reply]
Thank you!! How do I delete the brownie though? This is my first trial in deploying a smart contract. Could I try without the latter suggestion and then attempt after doing it if it fails or would this just cause further error?
